components/camel-platform-http-vertx/pom.xml

```diff
@@ -60,6 +60,11 @@
             <artifactId>vertx-auth-properties</artifactId>
             <version>${vertx-version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.vertx</groupId>
+            <artifactId>vertx-auth-jwt</artifactId>
```

Comment:
Do we need these `vertx-auth` dependencies added here? Or can they be added to `camel-platform-http-main` instead?

Reply:
@jamesnetherton thanks for noting that! Moved these dependencies to camel-platform-http-main